test/components/form_camera_info_test.rb

```diff
@@ -26,10 +26,10 @@ def test_renders_gps_info_with_all_values
     assert_includes(html, "exif_alt")
     assert_includes(html, "100")
 
-    # Check that values are separated by commas
-    assert_includes(html, "45.5231</span>, ")
-    assert_includes(html, "-122.6765</span>, ")
-    assert_includes(html, "100</span>")
+    # Wrappers should not have d-none class when values are present
```

Comment:
I don't understand this change.  What is the "d-none class"?  In what sense are the following assertions checking that some class is NOT present?  Why are we now just testing to see if a class is mentioned vs. numeric values?